jekyll/_cci2/persist-data.md

```diff
@@ -131,7 +145,12 @@ Charges apply when an organization has runner network egress beyond the included
 
```

Comment:
I can't add a comment to the section I wanted to so adding it below the paragraph here! THis comment is for line 144 

I think it would be good to clarify here the full scope of when charges apply: ....network egress **for workspaces and caches** beyond the included GB allotment..... **and going over your storage allowance** .... or something along those lines